site/src/pages/components/selectlist.mdx

```diff
@@ -59,18 +59,24 @@ Selectlist provides a variety of tools to help with styling, including pseudo-se
 
 ### Styling parts of the control
 
+Selectlist supports a variety of pseudo-elements to target its different parts:
+- `::button` - Targets the button slot.
+- `::listbox` - Targets the `<listbox>`.
+- `::marker` - Targets the dropdown arrow of the default supplied button.
+- `::selectedvalue` - Targets the `<selectedvalue>` element.
+
 One way to style the control to match your requirements is to use the CSS `::part()` pseudo-element to select the different parts within the control's anatomy that you wish to style.
```

Comment:
This is no longer the case and should be removed

Reply:
whoops, good catch. i removed this sentence